website/docs/reference/system_requirements.md

```diff
@@ -0,0 +1,89 @@
+---
+title: 'Spice.ai Open Source System Requirements'
+sidebar_label: 'Reference'
+sidebar_position: 30
+description: 'System requirements for running Spice.ai Open Source'
+pagination_prev: null
+pagination_next: null
+---
+
+This document outlines the system requirements for running Spice.ai Open Source. Ensure your environment meets these requirements to achieve optimal performance and stability. Note that resource requirements, particularly memory and storage, are highly dependent on workload and data. Specific recommendations are provided later in this document.
+
+## Operating Systems and Architectures
+
+Spice.ai supports the following operating systems and architectures:
+
+- **Linux**: Distributions as noted (x86_64, ARM)
+- **macOS**: macOS 12.0 (Monterey) or later (ARM/M-series only)
+- **Windows**: Windows 11 or later (x86, x64)
+
+## Linux Distribution Support
+
+Spice.ai supports the following Linux distributions:
+
+- **Ubuntu Server LTS**: 22.04 or later
+- **Amazon Linux**: 2 or later
```

Comment:
We should add Debian bookworm (12): https://www.debian.org/releases/bookworm/
That is what our Docker image uses: https://github.com/spiceai/spiceai/blob/trunk/Dockerfile#L29